.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     uses: Dwolla/jenkins-agents-workflow/.github/workflows/build-docker-image.yml@main
     with:
       IMAGE_NAME: dwolla/jenkins-agent-nvm
-      TAG_NAME: CORE_TAG_JDK8_JDK11
+      TAG_NAME: CORE_TAG
```

Comment:
More context in this [PR comment](https://github.com/Dwolla/jenkins-agent-docker-sbt/pull/24#discussion_r1148028936)